wherehows-web/app/templates/components/notifications/partials/stale-search-alert.hbs

```diff
@@ -0,0 +1,2 @@
+The WhereHows search index isn't up to date. Until that gets fixed, there are two workarounds: <br>
```

Comment:
The convention for `partials` in ember, is an underscore before the name of the partial
https://www.emberjs.com/api/ember/3.1/classes/Ember.Templates.helpers/methods/partial?anchor=partial

Reply:
Fixed